rollup.config.js

```diff
@@ -0,0 +1,32 @@
+const del = require('rollup-plugin-delete')
+const commonjs = require('@rollup/plugin-commonjs')
+
+const entries = [
+  './src/index.js',
+  './src/jest-globals.js',
+  './src/matchers.js',
+  './src/vitest.js',
+]
+
+module.exports = [
+  {
+    input: entries,
+    output: [
+      {
+        dir: 'dist',
+        entryFileNames: '[name].mjs',
+        chunkFileNames: '[name]-[hash].mjs',
+        format: 'esm',
+      },
+      {
+        dir: 'dist',
+        entryFileNames: '[name].js',
+        chunkFileNames: '[name]-[hash].js',
+        format: 'cjs',
+      },
+    ],
+    external: id =>
+      !id.startsWith('\0') && !id.startsWith('.') && !id.startsWith('/'),
```

Comment:
Out of curiosity, when would an import start with `\0`?

Reply:
It is a convention for rollup plugins that create virtual modules.  https://rollupjs.org/plugin-development/#conventions

We don't have any such plugins enabled here, so it could probably be removed if desired, but I kept it just to be safe.  Honestly I don't remember writing this originally, I may have copied it from somewhere else that suggested it.

Reply:
Thanks for the explanation. It's probably fine to keep in, I just haven't used rollup in several years and it stuck out to me. Esbuild has the convenient [`packages=external`](https://esbuild.github.io/api/#packages) option and Rollup doesn't appear to have anything similar.